cpu/cpuidle-latency.py

```diff
@@ -41,7 +41,7 @@ def setUp(self):
         '''
         sm = SoftwareManager()
         distro_name = distro.detect().name
-        deps = ['gcc', 'make']
+        deps = ['gcc', 'make', 'kernel-devel']
```

Comment:
is this change holds good for other than SLES distro as well

Reply:
Yes, changes are tested ok on Redhat system,
**Device:** Redhat
Avocado version:- 94.0
***********************************************************************
**Test Logs:**

`[root@alp11 cpu]# avocado run --test-runner runner cpuidle-latency.py -m cpuidle-latency.py.data/cpuidle-latency.yaml
JOB ID     : b783b76a5a3f17ddb89b3fc2a3ba005d17d88494
JOB LOG    : /root/avocado/job-results/job-2022-03-23T17.17-b783b76/job.log
 (1/1) cpuidle-latency.py:Cpuidle_latency.test;run-all_cpu-default-perf-2cbf: PASS (11.37 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2022-03-23T17.17-b783b76/results.html
JOB TIME   : 11.83 s
`